### ISSUE TITLE
Error when generate routes

### ISSUE THREAD

**Issue by guysolamour** [bkoski/slug#20]
I have installed the slug gem but I have an error with others models where I used the I'd column.  I got this error. 
Undefined method slug colum for.... 
How to fix it.  Thank you

**Comment by gsohn** [comment]
Same error :)

Any fix ?

**Comment by wbreeze** [comment]
Have reproduced.  All that was necessary was to add the gem. After adding the gem, there are test failures around `to_param`. Here is one:
```
Error:
ChoicesControllerTest#test_should_show_choice:
NoMethodError: undefined method `slug_column' for #<Choice:0x00007feba8cffc60>
    test/controllers/choices_controller_test.rb:41:in
   `block in <class:ChoicesControllerTest>'
```
The test looks like this:
```
 40   test "should show choice" do
 41     get :show, params: { id: @choice.to_param }
 42     assert_response :success
 43   end
```

**Comment by bkoski** [comment]
Just released v4.1.1 with the fixes from @wbreeze - @gsohn @guysolamour, let us know if you're still having trouble.
